source/vistas/core/plugins/stats.py

```diff
@@ -0,0 +1,83 @@
+import time
```

Comment:
Good with me, I just had it within `vistas/core/plugins` since it was plugin-specific, but I see that `vistas.core.stats` will be useful down the road.